ethereum/json-types/src/main/java/tech/pegasys/teku/ethereum/json/types/EthereumTypes.java

```diff
@@ -99,6 +107,34 @@ public class EthereumTypes {
           .example("phase0")
           .build();
 
+  public static final BooleanTypeHeaderDefinition ETH_HEADER_EXECUTION_PAYLOAD_BLINDED_TYPE =
+      new BooleanTypeHeaderDefinition(
+          HEADER_EXECUTION_PAYLOAD_BLINDED,
+          Optional.of(true),
+          "Required in response so client can deserialize returned json or ssz data to the correct object.");
+
+  public static final StringTypeHeaderDefinition<UInt256> ETH_HEADER_EXECUTION_PAYLOAD_VALUE_TYPE =
+      new StringTypeHeaderDefinition.Builder<UInt256>()
+          .title(HEADER_EXECUTION_PAYLOAD_VALUE)
+          .description(
+              "Execution payload value in Wei. Required in response so client can determine relative value of execution payloads.")
+          .formatter(value -> value.toBigInteger().toString(10))
+          .parser(value -> UInt256.valueOf(new BigInteger(value, 10)))
+          .example("1")
+          .required(true)
+          .build();
```

Comment:
i think the name here may be inconsistent... 
```
StringBasedPrimitiveTypeDefinition extends StringValueTypeDefinition
DeserializableTypeDefinition
SerializableTypeDefinition
```
`StringTypeHeaderDefintion` is kind of a different pattern..

I wonder if we want something like 

`StringBasedHeaderTypeDefinition` and `BooleanHeaderTypeDefinition` or something...

Reply:
truly the only difference is the way we serialize to the OpenAPI. But I could so with tiding up to follow a pattern.

Reply:
makes sense